src/mumble/Settings.h

```diff
@@ -258,6 +258,10 @@ struct Settings {
 	/// Name of the style to use from theme. @see Themes
 	QString themeStyleName;
 	
+	QFont qfFontLog;
+	QFont qfFontInput;
+	QFont qfFontTree;
```

Comment:
Documentation: A small Doxygen comment stating what these fonts are intended to be used for would improve readability